src/main/java/net/ltxprogrammer/changed/ChangedConfig.java

```diff
@@ -41,12 +39,20 @@ private interface AdditionalData {
     public static class Common {
         public final ForgeConfigSpec.ConfigValue<String> githubDomain;
         public final ForgeConfigSpec.ConfigValue<Boolean> displayPatronage;
+        public final ForgeConfigSpec.ConfigValue<Boolean> openOrigin ;
```

Comment:
Weird spacings and newlines all over the place.

Reply:
Can you clarify that?

Reply:
I'm being picky about the formatting of the code. On this line I commented on, there's a space in between `openOrigin` and the semicolon.